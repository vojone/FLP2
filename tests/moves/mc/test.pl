:- consult('../../../cube').

:- read_cube(C, _), move_m_ccw(C, _, _, RC), write_cube(RC).
