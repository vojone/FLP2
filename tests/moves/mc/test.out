143
416
719
414 111 282 333
121 222 353 444
131 222 323 444
636
636
626
